pkg/jsonmessage/jsonmessage.go

```diff
@@ -99,6 +99,7 @@ type JSONMessage struct {
 	ID              string        `json:"id,omitempty"`
 	From            string        `json:"from,omitempty"`
 	Time            int64         `json:"time,omitempty"`
+	TimeNano        int64         `json:"timeNano,omitempty"`
```

Comment:
It's present. And I thought we tried to not break API for ./pkg/?
On Sep 15, 2015 19:23, "David Calavera" <notifications@github.com> wrote:

> In pkg/jsonmessage/jsonmessage.go
> <https://github.com/docker/docker/pull/16317#discussion_r39578388>:
>
> > @@ -99,6 +99,7 @@ type JSONMessage struct {
> >  	ID              string        `json:"id,omitempty"`
> >  	From            string        `json:"from,omitempty"`
> >  	Time            int64         `json:"time,omitempty"`
> > +	TimeNano        int64         `json:"timeNano,omitempty"`
>
> but the existent field is not set anymore, is it? I don't know how I feel
> about a field that's never set.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/docker/pull/16317/files#r39578388>.
>